registry/pkg/adapter/adapter.go

```diff
@@ -268,6 +276,16 @@ func (a *registryAdapter) sendEvent(eventType string, desc *remote.Descriptor) e
 	return nil
 }
 
+func (a *registryAdapter) parseExtraAttributes() map[string]string {
+	result := map[string]string{}
+	keyValuePairs := strings.Split(a.env.Data, ",")
+	for _, keyValue := range keyValuePairs {
+		elements := strings.SplitN(keyValue, ":", 2)
+		result[elements[0]] = elements[1]
```

Comment:
we probably want to be more defensive about this.
an empty string would make this crash with an index-out-of-range error (as seen by @andrew-su and I because of an unrelated issue)